usr/sbin/pkcsconf/pkcsconf.c

```diff
@@ -862,8 +862,12 @@ void print_token_info(int slot_id, CK_TOKEN_INFO *TokenInfo)
 
     printf("\tSessions: %lu/%lu\n", TokenInfo->ulSessionCount,
            TokenInfo->ulMaxSessionCount);
-    printf("\tR/W Sessions: %lu/%lu\n", TokenInfo->ulRwSessionCount,
-           TokenInfo->ulMaxRwSessionCount);
+    if (TokenInfo->ulRwSessionCount == CK_UNAVAILABLE_INFORMATION)
```

Comment:
Why not also check for CK_UNAVAILABLE_INFORMATION and CK_EFFECTIVELY_INFINITE for other fields ?
```
CK_TOKEN_INFO Note: The fields ulMaxSessionCount, ulSessionCount, ulMaxRwSessionCount, 
ulRwSessionCount, ulTotalPublicMemory, ulFreePublicMemory, ulTotalPrivateMemory, 
and ulFreePrivateMemory can have the special value CK_UNAVAILABLE_INFORMATION, 
which means that the token and/or library is unable or unwilling to provide that information. 
In addition, the fields ulMaxSessionCount and ulMaxRwSessionCount can have the special value 
CK_EFFECTIVELY_INFINITE, which means that there is no practical limit on the number of sessions
(resp. R/W sessions) an application can have open with the token.

It is important to check these fields for these special values.  This is particularly true for 
CK_EFFECTIVELY_INFINITE, since an application seeing this value in the ulMaxSessionCount or 
ulMaxRwSessionCount field would otherwise conclude that it can’t open any sessions with the token, 
which is far from being the case.
```

Reply:
Good point. Will change this for the other fields, and also handle CK_EFFECTIVELY_INFINITE.